src/js/stores/export.js

```diff
@@ -181,6 +183,23 @@ define(function (require, exports, module) {
                 documentExports = DocumentExports.fromDescriptors(payload);
 
             this._documentExportsMap = this._documentExportsMap.set(documentID, documentExports);
+            this.emit("change");
```

Comment:
This is an unrelated bugfix. @mcilroyc, you may wish to confirm that adding this change event doesn't have any ill effects.